app/models/user.rb

```diff
@@ -24,6 +24,7 @@ class User < ApplicationRecord
   encrypts :pin
 
   has_many :orders, dependent: :destroy
+  has_many :packages, dependent: :destroy
```

Comment:
See more details here on [bi-directional ](https://guides.rubyonrails.org/association_basics.html#bi-directional-associations)assoaciations

```suggestion
  has_many :packages, inverse_of: "creator", dependent: :destroy
```